cnv/cnv_users_guide/cnv-ansible-modules.adoc

```diff
@@ -0,0 +1,15 @@
+[id="ansible-modules"]
+= Automating management tasks using Ansible
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-ansible-modules
+toc::[]
+
+Ansible is an automation tool used to configure systems, deploy software, and perform rolling updates.
+Ansible includes support for {ProductName}, and Ansible modules are available to allow you to automate
+cluster management tasks such as template, persistent volume claim and virtual machine operations.
+
+Ansible provides an alternative way of automating {ProductName} management compared to APIs and oc tool,
+and allows you to integrate with other Ansible modules.
+
+For more information about the Ansible modules available for {ProductName}, see link:https://docs.ansible.com/ansible/latest/modules/list_of_cloud_modules.html#kubevirt[Kubevirt modules]
```

Comment:
Generally we try to avoid "see this link." Maybe the previous sentence should read:

 "...allows you to integrate link:https://docs.ansible.com/ansible/latest/modules/list_of_cloud_modules.html#kubevirt[KubeVirt modules] with other Ansible modules." 

or something like that?

Reply:
Sure will update.

Reply:
@ousleyp What do you think about:

> Among other tools Ansible provides a way to automate {ProductName} management and allows to integrate link:https://docs.ansible.com/ansible/latest/modules/list_of_cloud_modules.html#kubevirt[KubeVirt modules] with other Ansible modules.

Reply:
> Ansible provides a way to automate {ProductName} management, which you can also accomplish by using the `oc` CLI tool or APIs. Ansible is unique because it allows you to integrate link:https://docs.ansible.com/ansible/latest/modules/list_of_cloud_modules.html#kubevirt[KubeVirt modules] with other Ansible modules.

What do you think of this? If it doesn't make sense, no worries.

Reply:
Looks good, will update